### DISCUSSION TITLE
QEditor inline mode

### DISCUSSION THREAD

**Discussion by Evertvdw** [quasarframework/quasar#15547] [Ideas / Proposals]
I think it would be nice to add an inline edit mode to the QEditor. I am currently using a custom implementation of this but I guess it would be something that would be useful to others as well. 

Example of what I mean: [TinyMce example](https://codepen.io/tinymce/pen/vYLqwKO)

The toolbar of the editor floats above the editor and remains in the viewport automatically when scrolling the top of the editor out of view.

**Comment by IlCallo** [comment]
That seems like a good idea to me

**Comment by capodisseny** [comment]
> I think it would be nice to add an inline edit mode to the QEditor. I am currently using a custom implementation of this but I guess it would be something that would be useful to others as well.
> 
> Example of what I mean: [TinyMce example](https://codepen.io/tinymce/pen/vYLqwKO)
> 
> The toolbar of the editor floats above the editor and remains in the viewport automatically when scrolling the top of the editor out of view.

Hi Evertvdw, how did you accomplish that? would you mind to share it? thanks!

**Reply by Evertvdw** [reply]
You can find the file I used back then here: https://gist.github.com/Evertvdw/b8405495e3ded373a3b10696177d11b3

You can use this as a guide to make your own version, I plucked the file straight out of our project back then so it contains all kind of references to other files in the project probably.

I moved on to use Tiptap since then, because I needed more flexibility then Quasar could offer.
